slack.go

```diff
@@ -411,6 +386,73 @@ func (b *Bot) Start(manager JobManager) error {
 		},
 	})
 
+	slack.Command("workflow-upgrade <name> <from_image_or_version_or_pr> <to_image_or_version_or_pr> <parameters>", &slacker.CommandDefinition{
+		Description: fmt.Sprintf("Run a custom upgrade using the requested workflow from an image or release or built PRs to a specified version/image/pr from https://amd64.ocp.releases.ci.openshift.org. "),
+		Handler: func(request slacker.Request, response slacker.ResponseWriter) {
+			user := request.Event().User
+			channel := request.Event().Channel
+			if !isDirectMessage(channel) {
+				response.Reply("this command is only accepted via direct message")
+				return
+			}
+
+			from, err := parseImageInput(request.StringParam("from_image_or_version_or_pr", ""))
+			if err != nil {
+				response.Reply(err.Error())
+				return
+			}
+			if len(from) == 0 {
+				response.Reply("you must specify initial release")
+				return
+			}
+
+			to, err := parseImageInput(request.StringParam("to_image_or_version_or_pr", ""))
+			if err != nil {
+				response.Reply(err.Error())
+				return
+			}
+			if len(from) == 0 {
+				response.Reply("you must specify the target release")
+				return
+			}
```

Comment:
Copy and paste problem.  This should check `to`

Reply:
Good catch, fixed